int32 botId
---
bool orderTaken
int32 deliveryLocationX
int32 deliveryLocationY